vpr/src/route/route_tree_timing.cpp

```diff
@@ -45,6 +52,7 @@ static t_linked_rt_edge* alloc_linked_rt_edge();
 static void free_linked_rt_edge(t_linked_rt_edge* rt_edge);
 
 static t_rt_node* add_subtree_to_route_tree(t_heap* hptr,
+                                            int target_pin,
```

Comment:
Rename target_pin to target_net_pin_index.  If you understand enough about what the routine does, write a comment here on what it does and what the input parameters are (if you don't feel confident in writing that, skip it).

Reply:
There is also already a comment at where the function is defined, which is in this file as well. I guess this is why none of the function declarations at the top of the file had comments to them. But if you think it's a good idea and a good convention to do so, I can add one in.

At where the function is defined, I moved the comment on top of the function and updated comments with the new parameter meaning.